Sources/Cards/OAImageCard.h

```diff
@@ -11,7 +11,7 @@
 
 @interface OAImageCard : OAAbstractCard
 
-@property (nonatomic, readonly) NSString *type;
+@property (nonatomic) NSString *type;
```

Comment:
Rather than breaking immutability, please override the fields in the subclass. (Example here:https://stackoverflow.com/a/41701165 and also this technique is used multiple times in the project (in OACollectionSettingsItem if I remember correctly).